internal/lvgen/generate.go

```diff
@@ -393,6 +399,138 @@ func procLink() {
 	}
 }
 
+// mapFlagTypes builds a map of the C types which appear to correspond to the
+// various flags fields in libvirt calls. Determining whether a type actually
+// corresponds to a set of flags is done by pattern matching the type name;
+// libvirt isn't completely consistent about the names of flag types, but they
+// all seem to have one of three suffixes, so that's what we look for here.
+//
+// This code uses the loader package to load the constants file generated by
+// c-for-go, which runs against libvirt's C sources. This file is generated by
+// 'go generate ./...' prior to the lvgen/ generator being run.
+func mapFlagTypes() map[string]ast.Expr {
+	pconf := loader.Config{}
+	f, err := pconf.ParseFile("../../const.gen.go", nil)
+	if err != nil {
+		fmt.Println("failed to read constants file: ", err)
```

Comment:
missing `return` ?

Reply:
Hmm, yes. I turned this into a panic - that error should never happen.